docs/experiments-plus/stratified-sampling.md

```diff
@@ -2,6 +2,7 @@
 title: Stratified Sampling
 sidebar_label: Stratified Sampling
 slug: /experiments-plus/stratified-sampling
+displayed_sidebar: cloud
```

Comment:
this already has the sidebar

https://docs.statsig.com/experiments-plus/stratified-sampling/

Reply:
removed it!